Sources/Spezi/Spezi/Spezi.swift

```diff
@@ -63,23 +72,51 @@ public actor Spezi<S: Standard>: AnySpezi, ObservableObject {
     init(
         standard: S,
         components: [any Component<S>],
-        _ logger: Logger = Logger(subsystem: "edu.stanford.spezi", category: "spezi")
+        _ logger: Logger = speziLogger
```

Comment:
We should ideally also explore the logger to some components using a knowledge source and maybe even a nice property wrapper that is encourage to use for logging in Spezi modules (apps should ideally have their own logging instance). But might be the content of a different PR if this goes too far.

Reply:
I like the idea of a `@Logger` property wrapper (using a dedicated property wrapper probably allows for better customization of a `subsystem` and a `category`).
I think that suites a separate PR a bit better. But we could store the global `speziLogger` using a module internal KnowledgeSource.